checks.d/network.py

```diff
@@ -4,6 +4,7 @@
 # stdlib
 import re
 import subprocess
+from subprocess import check_output
```

Comment:
Hi,
check_output is only available with Python 2.7 - It will work well with the embedded version that comes with Datadog:
sudo -u dd-agent dd-agent check network

Cheers